recipes/efs-setup.rb

```diff
@@ -61,4 +61,3 @@
   mount_point '/mnt/efs'
   action :umount
 end
-
```

Comment:
should you have a new line at end of file?

Reply:
There's still a new line. Just not two of them.